### PR TITLE
Feature/upload multiple dsym files

### PR DESCRIPTION
**Pull request by mo-abdelhafez**

### CODE REVIEW

File: lib/fastlane/plugin/instabug_official/actions/instabug_official_action.rb [hunk 1]
```diff
@@ -3,114 +3,125 @@
...
+            command = "curl #{endpoint} --write-out %{http_code} --silent --output /dev/null -F os=\"ios\" -F application_token=\"#{api_token}\" -F symbols_file="
+            
+            dsym_paths = (params[:dsym_array_paths] || []).uniq
+            UI.verbose "dsym_paths: " + dsym_paths.inspect
+            
+            
+            if dsym_paths.empty?
+                directory_name = fastlane_dsyms_filename
+                if directory_name.empty?
+                UI.error "Fastlane dSYMs file is not found! make sure your're using fastlane action [download_dsyms] to download your dSYMs from iTC"
```
Comment (Kmohamed) [inline lib/fastlane/plugin/instabug_official/actions/instabug_official_action.rb]:
```suggestion
                UI.error "Fastlane dSYMs file is not found! make sure you're using Fastlane action [download_dsyms] to download your dSYMs from App Store Connect"
```

File: lib/fastlane/plugin/instabug_official/actions/instabug_official_action.rb [hunk 2]
```diff
@@ -3,114 +3,125 @@
...
+    end
+    
+    def self.description
+    'upload dsyms to fastlane'
+end
 
-        UI.verbose curlCommand
-        return curlCommand if Helper.test?
+def self.authors
+['Karim_Mousa_89']
```
Comment (Kmohamed) [inline lib/fastlane/plugin/instabug_official/actions/instabug_official_action.rb]:
```suggestion
['Instabug Inc.']
```

File: lib/fastlane/plugin/instabug_official/actions/instabug_official_action.rb (hunk 1 above)
Comment (mo-abdelhafez) [reply lib/fastlane/plugin/instabug_official/actions/instabug_official_action.rb]: Done

File: lib/fastlane/plugin/instabug_official/actions/instabug_official_action.rb (hunk 2 above)
Comment (mo-abdelhafez) [reply lib/fastlane/plugin/instabug_official/actions/instabug_official_action.rb]: Done

**Comment by mo-abdelhafez** [comment]
@Kmohamed   I am just waiting to validate the full scenario.
What happens is, endpoint responses with `200Ok` but I cannot find the dSYMs on dashboard! 
I guess it's a pure backend issue. Maybe we can publish the plugin after validating the full scenario.

**Pull request merged**